scripts/tests/test_ci_lane_workflows.py

```
@@ -131,10 +134,44 @@ def test_lane_plans_are_bounded_platform_projections(self) -> None:
+        for output, lane in (
+            ("quality_lane_plan", "quality"),
+            ("website_lane_plan", "website"),
+        ):
+            with self.subTest(output=output):
+                match = re.search(
+                    rf"{output}=\$\(jq -ce '(.*?)' ci-plan\.json\)",
+                    action,
+                    re.DOTALL,
+                )
+                self.assertIsNotNone(match)
+                result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)